catalog/sync_test.go

```diff
@@ -13,9 +13,10 @@ import (
 )
 
 func TestSync(t *testing.T) {
-	if len(os.Getenv("INTTEST")) == 0 {
-		t.Skip("no int test env")
+	if os.Getenv("AWS_ACCESS_KEY_ID") == "" || os.Getenv("AWS_SECRET_ACCESS_KEY") == "" {
```

Comment:
Locally I don't have that set, because the aws sdk can get it from various places. Plus I don't want to run it every time.

Reply:
I removed this in favor of updating the documentation with some better requirements.